src/site/content/en/learn/html/focus/index.md

```diff
@@ -3,10 +3,228 @@ title: 'Focus'
 authors:
   - estelleweyl
 description: How to manage focus order in your HTML documents.
-date: 2022-09-08
-placeholder: true
+date: 2023-14-02
 tags:
   - html
 ---
 
-Coming soon!
\ No newline at end of file
+Interactive elements, including [form controls](/learn/html/forms/), [links](/learn/html/links/), and buttons, are by
+default focusable and tabbable. Tabbable elements are part of the document's sequential focus navigation order. Other elements
+are inert, meaning they are not interactive. With HTML attributes, it is possible to make interactive elements inert and to
+make inert elements interactive.
+
+{% Aside %}
+For usability reasons, always ensure that the user knows which element has focus. Include CSS [`:focus`](https://developer.mozilla.org/docs/Web/CSS/:focus),
+[`:focus-visible`](https://developer.mozilla.org/docs/Web/CSS/:focus-visible), and, optionally, [`:focus-within`](https://developer.mozilla.org/docs/Web/CSS/:focus-within)
+styles. This is so important: there are Learn sections devoted to focus styles in both the CSS and Accessibility learning paths.
+{% endAside %}
+
+By default, the navigation focus order is the same as the visual order, which is the source code order. There are HTML
+attributes that can alter this order and CSS properties that can alter the visual order of content. Changing the tabbing
+order with HTML or visual rendering order with CSS can harm user experience.
+
+Don't alter the perceived and actual tabbing order with CSS and HTML. As the following two examples demonstrate, tab orders
+that differ from the visually expected order are confusing to users and bad for user experience.
+
+In this example, the value of the [`tabindex`](https://developer.mozilla.org/docs/Web/HTML/Global_attributes/tabindex) attribute
+has made the tab order chaotic:
+
+{% Codepen {
+user: 'web-dot-dev',
+id: 'GRXKqGz',
+tab: 'result'
+} %}
+
+In this example, CSS has created a divergence between the tabbing order and the visual order of the content:
+
+{% Codepen {
```

Comment:
there is an issue with my original codepen. I blame the cat! Please remove the[;"]